kolibri/plugins/learn/assets/src/views/AlsoInThis.vue

```diff
@@ -24,31 +24,34 @@
           :kind="content.learning_activities"
         />
         <KIcon v-else class="topic-icon" icon="topic" />
-        <div class="content-meta" :style="{ top: (content.duration ? '-8px' : '8px') }">
-          <TextTruncator
-            :text="content.title"
-            :maxHeight="24"
-            :style="{ marginTop: (content.duration ? '8px' : '0px') }"
-          />
-          <TimeDuration
-            v-if="content.duration"
-            :style="{
-              color: $themeTokens.annotation,
-              top: '16px',
-              position: 'relative'
-            }"
-            :seconds="content.duration"
-          />
-        </div>
-        <div class="progress">
-          <KIcon
-            v-if="content.progress === 1"
-            icon="star"
-            class="mastered-icon"
-            :style="{ fill: $themeTokens.mastered }"
-          />
-          <ProgressBar v-else :contentNode="content" class="bar" />
+
+        <div class="content-meta">
+          <div class="text-and-time">
+            <TextTruncator
+              class="content-title"
+              :text="content.title"
+              :maxHeight="72"
+            />
+            <TimeDuration
+              v-if="content.duration"
+              class="time-duration"
+              :style="{
+                color: $themeTokens.annotation,
+              }"
+              :seconds="content.duration"
+            />
+          </div>
+          <div class="progress">
+            <KIcon
```

Comment:
This `KIcon` is also inside the `ProgressBar` component - is there a styling or layout reason to not to defer to it here? If so, would be good to have a code comment to explain.